src/components/camera/camera.tsx

```diff
@@ -1,4 +1,4 @@
-import { h, Component, Element, Prop, State } from '@stencil/core';
+import { h, Component, Element, Prop, State, Build } from '@stencil/core';
```

Comment:
As part of the de-risk work in FW-4065 it was noted that we may want to opt-in to legacy browser support. Is this something we need? We might want to check to see what the browser support for this package is.

Reply:
Addressed here: https://github.com/ionic-team/pwa-elements/pull/118/commits/4bb84706e3618a269e0002fb12a18b2a493cb5f5